src/App.tsx

```diff
@@ -10,33 +11,21 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setTabId] = useState('tab-1');
```

Comment:
Better use `tabs[0].id` because this way it would work with different data. If id would change your solution would need some changes.
```suggestion
  const [selectedTabId, setTabId] = useState(tabs[0].id);
```